de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/api/ShipmentScheduleWithHU.java

```diff
@@ -144,6 +149,10 @@ private ShipmentScheduleWithHU(
 		this.vhu = shipmentScheduleAlloc.getVHU_ID() > 0 ? shipmentScheduleAlloc.getVHU() : null;
 		this.tuHU = shipmentScheduleAlloc.getM_TU_HU_ID() > 0 ? shipmentScheduleAlloc.getM_TU_HU() : null;
 		this.luHU = shipmentScheduleAlloc.getM_LU_HU_ID() > 0 ? shipmentScheduleAlloc.getM_LU_HU() : null;
+
+		this.setForPickedQty(isForPicked);
+		this.setQtyTypeToUse(qtyTypeToUse);
```

Comment:
`this.setQtyTypeToUse(qtyTypeToUse);` doesn't have any effect right now because the `qtyTypeToUse` is `this.qtyTypeToUse`